Resources/Prototypes/Roles/Jobs/Security/detective.yml

```diff
@@ -7,12 +7,14 @@
   - !type:DepartmentTimeRequirement
     department: Security
     time: 54000 # 15 hours
+  - !type:DepartmentTimeRequirement
+      department: Civilian
+      time: 18000 # 5 hours

```

Comment:
i don't get what this is for and it seems strange

Reply:
seems the same as the playtime required for borgs. in this case, you need 5 hours of playtime to take up the role of this specific department.